src/common.tests/Logging/MacAddressHash/MacInformationProviderTests.cs

```diff
@@ -0,0 +1,37 @@
+﻿using FakeItEasy;
+using Microsoft.BridgeToKubernetes.Common.IO;
+using Microsoft.BridgeToKubernetes.Common.Logging.MacAddressHash;
+using Microsoft.BridgeToKubernetes.Common.PersistentProperyBag;
+using Microsoft.BridgeToKubernetes.TestHelpers;
+using System;
+using Xunit;
+
+namespace Microsoft.BridgeToKubernetes.Common.Tests.Logging.MacAddressHash
+{
+    public class MacInformationProviderTests : TestsBase
+    {
+        [Fact]
+        public void GetMacAddressHashOnWindows()
```

Comment:
@Eneuman can you add one more test for Mac/Linux for mach address hash please ? Thank you. I have shared a sample output for ```ifconfig```, please check.

Reply:
@hsubramanianaks I have added a test for Linux :)